packages/component/src/SendBox/TextBox.js

```diff
@@ -69,7 +69,7 @@ const TextBox = ({ className, disabled, language, onChange, onKeyPress, onSubmit
           {({ sendFocusRef }) =>
             !sendBoxTextWrap ? (
               <input
-                aria-label={typeYourMessageString}
+                aria-label={'Sendbox'}
```

Comment:
Does this need to be localized?

Reply:
And we should use `aria-label="Sendbox"` rather than `aria-label={'Sendbox'}` for strings.

Reply:
Thanks, localization added (ping)